File: CPP/Harddrive.cpp
#include "Komponen.cpp"

class Harddrive : public Komponen {
private:
    int kapasitas;
public:
    Harddrive() {}
    Harddrive(string nama, int kapasitas) : Komponen(nama) {
        this->kapasitas = kapasitas;
    }
    void tampil() {
        cout << "Harddrive : " << nama << " (" << kapasitas << " GB) " << endl;
    }
};

File: Python/Harddrive.py
from Komponen import Komponen

class Harddrive(Komponen):
    def __init__(self, nama, kapasitas):
        super().__init__(nama)
        self.kapasitas = kapasitas
    
    def get_info(self):
        return f"{self.nama} ({self.kapasitas} GB)"